compute/cloud-client/src/test/java/compute/InstanceOperationsIT.java

```diff
@@ -204,14 +226,19 @@ public void testEncryptedInstanceOperations()
   @Test
   public void testCloneEncryptedDisk()
       throws IOException, ExecutionException, InterruptedException, TimeoutException {
-    Assert.assertEquals(Util.getInstanceStatus(PROJECT_ID, ZONE, MACHINE_NAME_ENCRYPTED),
-        "RUNNING");
+    ByteArrayOutputStream stdOut = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(stdOut));
+
+    Assert.assertEquals("RUNNING",
+            Util.getInstanceStatus(PROJECT_ID, ZONE, MACHINE_NAME_ENCRYPTED));
```

Comment:
Do we need this assertion? It is prone to failure as the state might differ on each run.

Reply:
This is your test and not related to this PR. But if you think it can improve this test, I'll delete this assertion.

Reply:
Yes, please delete. Thank you.